proto/buf/alpha/audit/v1alpha1/envelope.proto

```diff
@@ -614,6 +615,10 @@ message UserActor {
   string subject = 5;
 }
 
+message MachineActor {
```

Comment:
I think my only worry is we use the term "machine" for things like "machine token" and those belong to users. Maybe `ServiceActor` would be more accurate? No strong opinion though..